pkg/wait/wait.go

```diff
@@ -16,69 +16,76 @@ import (
 // ForPodsRunning waits for pods to be running
 func ForPodsRunning(clName string, c kubernetes.Interface, namespace, selector string, replicas int) error {
 	ctx := context.Background()
-	log.Debugf("Waiting for pods to be running. label: %q, namespace: %q, replicas: %v, duration: %v, *types: %s.", selector, namespace, replicas, defaults.WaitDurationResources, clName)
+	log.Infof("Waiting up to %v for pods running with label %q, namespace %q, replicas %v in cluster %q ...", defaults.WaitDurationResources, selector, namespace, replicas, clName)
 	podsContext, cancel := context.WithTimeout(ctx, defaults.WaitDurationResources)
 	wait.Until(func() {
 		podList, err := c.CoreV1().Pods(namespace).List(metav1.ListOptions{
 			LabelSelector: selector,
 			FieldSelector: "status.phase=Running",
 		})
-		if err == nil && len(podList.Items) == replicas {
-			log.Infof("✔ All pods with label: %q are running for %s.", selector, clName)
+
+		if err != nil {
+			log.Errorf("Error listing pods for label %q, namespace %q in cluster %q: %v", selector, namespace, clName, err)
+		} else if len(podList.Items) == replicas {
+			log.Infof("✔ All pods with label %q in namespace %q are running in cluster %q.", selector, namespace, clName)
 			cancel()
 		} else {
-			log.Debugf("Still waiting for pods. label: %q, namespace: %q, replicas: %v, cluster: %s.", selector, namespace, replicas, clName)
+			log.Infof("Still waiting for pods with label %q, namespace %q, replicas %v in cluster: %q.", selector, namespace, replicas, clName)
 		}
-	}, 15*time.Second, podsContext.Done())
+	}, 5*time.Second, podsContext.Done())
 
 	err := podsContext.Err()
 	if err != nil && err != context.Canceled {
-		return errors.Wrap(err, "Error waiting for pods to be running.")
+		return errors.Wrap(err, "Error waiting for pods running.")
```

Comment:
I liked it better with "to be", but very non-blocker.

Reply:
you're right - not sure what I was thinking - changed it back.